circle.yml

```diff
@@ -37,9 +37,8 @@ dependencies:
       else
         echo "Conda already set up.";
       fi
-    - conda install -n circleenv --yes numpy scipy scikit-learn matplotlib sphinx=1.4.1 pillow six IPython pandas;
-    - conda install -n circleenv --yes traits=4.6.0 traitsui=5.1.0;
-    - conda install -n circleenv --yes mayavi;
+    - conda install -n circleenv --yes numpy scipy scikit-learn matplotlib sphinx=1.4.1 pillow six IPython pandas apptools=4.4.0 envisage=4.5.1 pyface=5.1.0 traits=4.6.0 traitsui=5.1.0 vtk=5.10.1;
+    - pip install mayavi==4.4.4;
```

Comment:
Can we install `mayavi` with conda but then `pip --upgrade` whatever package it gets errors on?